tests/cases/fourslash/codeFixAddConvertToUnknownForNonOverlappingTypes_all.ts

```diff
@@ -0,0 +1,22 @@
+/// <reference path='fourslash.ts' />
+
+////class C {
+////    const s1 = 1 as string;
+////    const o1 = s + " word" as object;
+////
+////    const s2 = <string>2;
+////    const o2 = <object>s2;
+////}
+
+verify.codeFixAll({
+    fixId: "addConvertToUnknownForNonOverlappingTypes",
+    fixAllDescription: "Add 'unknown' to all conversions of non-overlapping types",
+    newFileContent:
+`class C {
+    const s1 = 1 as unknown as string;
+    const o1 = s + " word" as unknown as object;
+
+    const s2 = <string><unknown>2;
+    const o2 = <object>s2;

```

Comment:
This should get the fix too, but doesn't because of parse errors. Fixed by removing the `class C { ... }` wrapper.